### DISCUSSION TITLE
Show git repo name, not just branch name on prompt

### DISCUSSION THREAD

**Discussion by o6uoq** [starship/starship#2938] [Q&A]
👋🏼 

I cannot find anywhere obvious where I can display the git repo name/slug to the prompt.

I'm often finding myself having to type `pwd` to figure out which repo I am in (which I need _with_ branch name).

Please and thank you in advance! 🙏🏼

**Comment by bradleyfrank** [comment]
I second this, it would be a super helpful feature!

**Comment by o6uoq** [comment]
Hello Starship! Is there an update to this discussion? Please and thank you! 🙏🏼

**Comment by o6uoq** [comment]
I think [this](https://github.com/starship/starship/issues/2419) is somewhat similar. Does anyone else have other examples or similar use cases which would be bundled together under some sort of `git_repo` type function?
